homeassistant/components/yessssms/notify.py

```diff
@@ -36,17 +36,18 @@ def get_service(hass, config, discovery_info=None):
         _LOGGER.error("Unknown provider: %s", ex)
         return None
     try:
-        if yesss.login_data_valid():
-            _LOGGER.info("Login data for '%s' valid.", yesss.get_provider())
-        else:
+        if not yesss.login_data_valid():
             _LOGGER.error(
-                "Login data is not valid! Please double check your login data at %s.",
+                "Login data is not valid! Please double check your login data at %s",
                 yesss.get_login_url(),
             )
             return None
+
+        _LOGGER.info("Login data for '%s' valid", yesss.get_provider())
```

Comment:
Maybe we can just remove this message and assume it's valid if we don't say otherwise? There's no important data to log anyway.

Reply:
we could. It however is a good indicator that everything is ok and will work.
It is an info message... so not very prominent.

Reply:
Then please change to debug instead. We try to avoid info logging.

Reply:
ok.